homeassistant/components/hassio/__init__.py

```diff
@@ -204,6 +205,10 @@ class APIEndpointSettings(NamedTuple):
     ),
 }
 
+HARDWARE_INTEGRATIONS = {
+    "rpi": "raspberry_pi",
```

Comment:
`raspberry_pi` doesn't match the `raspberrypi` domain. We probably need to change either place to match.

Reply:
Shouldn't the manifest provide details on the type of integration?

`"integration_type": "hardware"`

Reply:
perhaps, but something would still need to translate the board type to an integration

Reply:
I think that should be the responsibility of the integration (e.g., using a discovery kinda solution).